### ISSUE TITLE
Objects disappear when trying to create a craft

### ISSUE THREAD

**Issue by AsuraZD** [WolfyScript/CustomCrafting#347]
### What happened?

The objects disappear when trying to create any type of crafting, that is, with all the tables, as shown in the follo

https://github.com/WolfyScript/CustomCrafting/assets/113893916/3fb68f92-06ae-4745-9b16-e8be1eeb56f5

wing video

### How can the bug be reproduced?

I already reported this error on discord to which they responded that wolfy utilities beta 7 should have solved this problem, but it is not, only the error no longer appears in the console but the items continue to disappear, please solve this soon since I use customcrafting on my server and I can't advance on it because of this bug

### Expected Behaviour?

Please fix it since this error continues and members of my server want to play but I can't launch it because of this bug

### Plugin version

customcrafting-spigot-4.16.8.5.jar, wolfyutils-spigot-4.16.15-beta.7.jar

### Server software

paper 1.19.4

### Submit issue

- [X] read the Wiki and especially the FaQ, and made sure there is no fix to your issue,
- [X] searched for and ensured there isn't already an open issue regarding this,
- [X] are running the latest version of your server software (Spigot, Paper, etc.), and plugins (WolfyUtilities, CustomCrafting).

**Comment by WolfyScript** [comment]
Yep I thought I fixed it. Are there any errors in the console when placing the item into the GUI?

**Comment by AsuraZD** [comment]
none, it just disappears...
